.ruby-version

```diff
@@ -1 +0,0 @@
-2.2.2
```

Comment:
think it conflicts if we ever decide to test multiple ruby versions. btw, this is useful for that in the future: https://github.com/thoughtbot/appraisal